src/components/menu/menu.spec.tsx

```diff
@@ -280,13 +247,17 @@ describe("Menu", () => {
         openSubmenu(wrapper, 0);
 
         act(() => {
-          menuItem.props().onClick();
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          menuItem.props().onClick!({} as React.MouseEvent<HTMLButtonElement>);
```

Comment:
What do you think of doing this so we are avoiding ignoring typescript and mimicking the user's action?
```suggestion
          menuItem.simulate("click");
```

Reply:
The tests fail when I update it to use simulate, as these are test files I personally don't think it's an issue to use a non-null assertion here as using an `if` will also require me to add a lint ignore statement